src/constructorio-id.js

```diff
@@ -7,19 +7,32 @@
     var defaults = {
       user_agent: null,
       persist: true,
-      cookie_name: 'ConstructorioID_client_id',
+      cookie_name_client_id: 'ConstructorioID_client_id',
```

Comment:
What do you think of changing from using cookie_name/local_name as a prefix to a suffix (i.e. client_id_cookie_name/client_id_local_name so that it reads a little better and matches up with client_id_storage_location?

Reply:
I was thinking about this as well - sounds good to me. @rubikzube any thoughts?